Moople.js

```diff
@@ -487,27 +458,66 @@ function moopleText(txt, idd, fnt, clr, xpos, ypos)
 	return t;
 }
 
-function isLetter(key) {
-  return key >= 'a' || key <= 'z' || key >= 'A' || key <= 'Z';
-}
-
 moopleGame.prototype.handleKeyboard = function()
 {
-  var _this = this;
-
-  $(document).on('keydown', function(e){
-    if (isLetter(e.key)) {
-      var property = e.key.toLowerCase() + 'IsDown';
-      _this[property] = true;
-    }
-  })
-  .on('keyup', function(e){
-    if (isLetter(e.key)) {
-      var property = e.key.toLowerCase() + 'IsDown';
-      _this[property] = false;
-    }
-  });
+	$(document).on('keydown', function(e){

```

Comment:
The master branch was updated while I was writing and I didn't want to rebase, hence the end of the pull request description saying "feel free to not merge but just take the setup changes."